packages/components/src/assets/copy-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 1H11.1765C11.7288 1 12.1765 1.44771 12.1765 2V11.1765C12.1765 11.7288 11.7288 12.1765 11.1765 12.1765H2C1.44771 12.1765 1 11.7288 1 11.1765V2C1 1.44771 1.44771 1 2 1ZM0 2C0 0.895428 0.895431 0 2 0H11.1765C12.281 0 13.1765 0.895431 13.1765 2V2.82353H14C15.1046 2.82353 16 3.71896 16 4.82353V14C16 15.1046 15.1046 16 14 16H4.82353C3.71896 16 2.82353 15.1046 2.82353 14V13.1765H3.82353V14C3.82353 14.5523 4.27124 15 4.82353 15H14C14.5523 15 15 14.5523 15 14V4.82353C15 4.27124 14.5523 3.82353 14 3.82353H13.1765V11.1765C13.1765 12.281 12.281 13.1765 11.1765 13.1765H2C0.895428 13.1765 0 12.281 0 11.1765V2Z" fill="#3E87D7"/>
```

Comment:
It's useful if the SVG and its children contain no `fill` properties because it allows us to easily change the color via CSS.

Reply:
Okay, good to know, I removed the `fill` properties.